Entity/LineTimetable.php

```diff
@@ -45,26 +46,26 @@ public function getId()
     }
 
     /**
-     * Set lineConfig
+     * Get lineConfig
      *
-     * @param LineConfig $lineConfig
-     * @return LineTimetable
+     * @return LineConfig
      */
-    public function setLineConfig(LineConfig $lineConfig = null)
+    public function getLineConfig()
     {
-        $this->lineConfig = $lineConfig;
-
-        return $this;
+        return $this->lineConfig;
     }
 
     /**
-     * Get lineConfig
+     * Set lineConfig
      *
-     * @return LineConfig
+     * @param LineConfig $lineConfig
+     * @return Timetable
      */
-    public function getLineConfig()
+    public function setLineConfig(LineConfig $lineConfig = null)
```

Comment:
I have a question about lineConfig anyway, I see in stopTimetable entity : 

https://github.com/Tisseo/MttBundle/blob/feature-timetable-entity/Entity/StopTimetable.php#L40-L43

But the lineConfig is persisted because it's a FK in database. So why is this comment there ?